ruby/rental.rb

```diff
@@ -1,11 +1,14 @@
+require_relative 'person'
+require_relative 'book'
+
 class Rental
   attr_accessor :date, :book, :person
 
   def initialize(date, book, person)
     @date = date
     @book = book
-    book.rentals(self)
+    book.rentals << self
```

Comment:
- [ ] Good job so far but I cannot still create a rental yet. I think the reason is that you were creating your has_many relationship with rentals i.e `book.rentals << self` which does not exist because you initialized your rentals as` rental ` and also pushing to the `rental `array which was supposed to be `rentals` in your Book class (book.rb) so it can't find `rentals` attribute as it can only find `rental` attribute.

I think you should debug from here 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/232250519-a3797dd6-58e3-454c-9cac-c152045945e9.png)

This is the error I am currently getting 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/232250549-5401df8e-a1c6-4652-b758-18dac63920a9.png)